_example/animation/game.js

```diff
@@ -0,0 +1,114 @@
+/* global Cervus */
+
+const get_scaling = (out, mat) => {
+  let m11 = mat[0];
+  let m12 = mat[1];
+  let m13 = mat[2];
+  let m21 = mat[4];
+  let m22 = mat[5];
+  let m23 = mat[6];
+  let m31 = mat[8];
+  let m32 = mat[9];
+  let m33 = mat[10];
+  out[0] = Math.sqrt(m11 * m11 + m12 * m12 + m13 * m13);
+  out[1] = Math.sqrt(m21 * m21 + m22 * m22 + m23 * m23);
+  out[2] = Math.sqrt(m31 * m31 + m32 * m32 + m33 * m33);
+  return out;
+};
+
+
+const game = new Cervus.core.Game({
+  width: window.innerWidth,
+  height: window.innerHeight,
+  light_intensity: 0.4,
+  // clear_color: 'ff00ff'
+  // fps: 10
+});
+
+game.camera.position = [0, 1, 2];
+game.camera.rotate_rl(Math.PI);
+game.camera.keyboard_controlled = true;
+// game.camera.mouse_controlled = true;
+
+const frames = [];
+const srames = [];
+
+let position, scale;
+
+fetch('models/ptak.json')
+.then((data) => {
+  return data.json();
+})
+.then((json) => {
+  console.log(json);
+  json.rootnode.children.forEach((child) => {
+    if (!child.meshes) {
+      return;
+    }
+
+    child.meshes.forEach((mesh_i, i) => {
+      frames.push({
+        vertices: json.meshes[mesh_i].vertices,
+        indices: [].concat.apply([], json.meshes[mesh_i].faces),
+        normals: json.meshes[mesh_i].normals
+      });
+      srames.push({
+        vertices: json.meshes[mesh_i].vertices,
+        indices: [].concat.apply([], json.meshes[mesh_i].faces),
+        normals: json.meshes[mesh_i].normals
+      });
```

Comment:
This looks like you could first create the `{vertices, indices, normals}` object and then push it to both `frames` and `srames`. What are `srames` by the way? I only see them used in a code that's commented out.